pay-api/tests/unit/api/test_eft_short_names.py

```diff
@@ -274,14 +275,19 @@ def assert_short_name_summary(result_dict: dict,
                               short_name: EFTShortnamesModel,
                               transaction: EFTTransactionModel,
                               expected_credits_remaining: Decimal,
-                              expected_linked_accounts_count: int):
+                              expected_linked_accounts_count: int,
+                              shortname_refund: EFTRefundModel = None):
     """Assert short name summary result."""
     date_format = '%Y-%m-%dT%H:%M:%S'
     assert result_dict['id'] == short_name.id
     assert result_dict['shortName'] == short_name.short_name
     assert result_dict['creditsRemaining'] == expected_credits_remaining
     assert result_dict['linkedAccountsCount'] == expected_linked_accounts_count
     assert datetime.strptime(result_dict['lastPaymentReceivedDate'], date_format) == transaction.deposit_date
+    if shortname_refund is None:
```

Comment:
`assert shortname_refund is None or result_dict['refundStatus'] == shortname_refund.status`